<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DSAReader</class>
 <widget class="QWidget" name="DSAReader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>263</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="LoadBtn">
      <property name="text">
       <string>Load</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Data</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QmitkDataStorageComboBox" name="DataSelector"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="ConfirmOriginFrameBtn">
        <property name="text">
         <string>Origin</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="OriginFrameLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ConfirmTargetFrameBtn">
        <property name="text">
         <string>Target</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="TargetFrameLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="CutBtn">
        <property name="text">
         <string>Cut</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="PreRegisterCB">
        <property name="text">
         <string>Pre Register</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="AnimationCB">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Animation</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="CompareBtn">
      <property name="text">
       <string>Compare Histogram</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="CompareResut"/>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QmitkDataStorageComboBox</class>
   <extends>QComboBox</extends>
   <header>QmitkDataStorageComboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
